src/factories/series/Area.ts

```diff
@@ -1,39 +1,69 @@
 module n3Charts.Factory.Series {
   'use strict';
 
-  export class Area extends Line {
-    containerClassName = 'area-data';
-    itemsClassName = 'area-series';
-    itemClassName = 'area';
+  export class Area extends Utils.SeriesFactory {
+
+    static type: string = Utils.Options.SERIES_TYPES.AREA;
```

Comment:
I liked the idea, that all types are defined in the `Options` class - I only wanted to avoid having to call `Utils.Options.SERIES_TYPES.AREA` inside a function.